src/main/java/digital/slovensko/autogram/core/eforms/EFormUtils.java

```diff
@@ -220,23 +222,33 @@ public static String getNamespaceFromEformXml(Document xml) {
         return null;
     }
 
-    public static byte[] getResource(String url) {
+    public static byte[] getResource(String url) throws ServiceUnavailableException {
         var offlineFileLoader = new FileCacheDataLoader();
         offlineFileLoader.setCacheExpirationTime(21600000);  // 6 hours
         offlineFileLoader.setDataLoader(new CommonsDataLoader());
 
         DSSDocument xsltDoc;
         try {
             xsltDoc = offlineFileLoader.getDocument(url);
+        } catch (DSSExternalResourceException e) {
+            var matcher = Pattern.compile("HTTP status code : (\\d{3})").matcher(e.getCause().getMessage());
+            if (!matcher.find())
+                return null;
+
+            var statusCode = matcher.group(1);
```

Comment:
táto premenná je zbytocná, ak sa používa iba raz a je to taký krátky výraz - vieme to použiť rovno.

Reply:
Jasné, to by sa možno potom hodilo ešte pozrieť a refactornúť celý kód (samozrejme nie tu) keďže myslím, že som takéto veci videl viackrát 😅👍